### DISCUSSION TITLE
MQTT implementation

### DISCUSSION THREAD

**Discussion by axi92** [upptime/upptime#843] [Ideas]
Is there a plan to implement a status on mqtt?
I saw that ws is already supportet. https://github.com/upptime/uptime-monitor/blob/master/src/update.ts#L178

Is there a way to work on that uptime-monitor repo local without github actions?
I have never developed a github action, thats why I am asking.
If I can test my changes local I am maybe able to provide a PR.

**Comment by AnandChowdhary** [comment]
This is not currently supported, but if you can recommend a good npm package that supports mqtt, we can use that to add this :)

**Reply by axi92** [reply]
There is the standard https://www.npmjs.com/package/mqtt 
Or we could provide you a package that behaves the same way as the tcp-ping package that you already use? So you have the same json result.
